@remirror/editor-social/src/social-types.ts

```diff
@@ -79,13 +79,16 @@ export type MatchName = 'at' | 'tag';
 export type MentionState = AtMentionState | HashMentionState;
 
 export interface UserData {
-  uid: string;
+  id?: string;
```

Comment:
I changed `uid` to `id`; this might be a breaking change (?). `id` for identifier seems more standard; `uid` is often interpretted as `user ID` rather than `unique ID`, so I thought since "identifier" already implies uniqueness `id` would be sufficient.